pkg/apiserver/event/event.go

```diff
@@ -47,11 +51,45 @@ func NewService(
 
 // Register mounts our HTTP handler on the mux.
 func Register(r *gin.RouterGroup, s *Service) {
-	endpoint := r.Group("/event")
+	endpoint := r.Group("/events")
 
 	// TODO: add more api handlers
 	endpoint.GET("/all", s.listEvents)
```

Comment:
We can use "endpoint.GET("l", s.listEvents)" here if this makes sense to you

Reply:
Why is it "l"?

Reply:
sorry a typo

Reply:
There is no modification here, I don't quite understand what you mean.